### DISCUSSION TITLE
CSS errors with Diamond 5.0.0

### DISCUSSION THREAD

**Discussion by vitsin** [primefaces/community#2555] [PrimeFaces Templates] [answered]
After purchasing Diamond 5.0.0 I wanted to create a simple landing page similar to [this](https://primefaces.org/diamond/landing.html)

Issues:
1.althought landing page displays, there are many errors in the Firefox's Inspector, something like
`downloadable font: download failed (font-family: "Nunito" style:normal weight:700 stretch:100 src index:2): status=2147746065 source: http://localhost:8080/jakarta.faces.resource/fonts/nunito-v12-latin-ext_latin-700.woff2`

![diamond500-errors](https://github.com/user-attachments/assets/20a7111e-ec41-4c1f-be98-68bd6c6e3eba)

2.I've tried than to re-compile SASS with code from Diamond, like it says [here](https://primefaces.org/diamond/documentation.xhtml)
but keep getting this error:
`
Deprecation Warning on line 230, column 36 of src\main\webapp\resources\sass\variables\theme\_theme_dim.scss:
!default should only be written once for each variable.
This will be an error in Dart Sass 2.0.0.
    ╷
230 │ $radiobuttonIconSize:12px !default !default;
    │                                                             ^^^^^^^^
    ╵
`
It looks like a 2nd "!default" need to be removed?

Anyway,
to re-create the issue I've made simple Jakarta and test based project 
[primefaces-test-jakarta.zip](https://github.com/user-attachments/files/16698868/primefaces-test-jakarta.zip)

Note: this ZIP does not include actual real Diamond CSS files (they are ~15Mb unzipped) from diamond-layout-5.0.0.zip, so one need to put them all under primefaces-test-jakarta/src/main/webapp/resources/[diamond CSS files/dirs here]

To start the project use [this instructions](https://github.com/primefaces/primefaces-test/) and than simply go to http://localhost:8080

regards,
--vs

**Comment by melloware** [comment, marked as answer]
@vitsin if you want to shoot me a DM on the Discord channel i have a local Diamond I can help you with all fixed up and working with PF14.
